### PR TITLE
Fire's update of alien hardsuit tweaks

### PR DESCRIPTION
**Pull request by Fire-and-Glory**
TL;DR mostly a bunch of minor tweaks to alien hardsuits aside from the Unathi stealth rig which I made to look more like the human stealth rig and less like the Unathi ninja suit.

+  - tweak: "The Tajaran variant of the AMI has been modified following complaints of their feet being 'literrrally not shaped for the boots'."
+  - tweak: "The Unathi stealth rig looks a little less like a ninja suit and a bit more like the stealth rig. (didn't have the heart to do the Tajarn&Skrell rigs)"
+  - tweak: "Low-profile changes have been made to a couple other alien RIGsuits."

### CODE REVIEW

**Comment by Arrow768** [comment]
Could you upload a few images showing the changes ?

**Comment by Alberyk** [comment]
icons/mob/species/unathi/suit.dmi are tajaran sprites instead of unathi ones.

**Comment by Fire-and-Glory** [comment]
Now that I am less tired, I will describe and show the changes.

Heavy Asset Protection helmets now all have narrower visors, antenna, and a transparent visor in their inactive states, so they now represent the human HAP helmet instead of the human ERT helmet.

All Tajaran ERT helmets HAPT included no longer permit the edge of the Tajaran's face to poke out of the helmet.

I may have made the Tajaran-Industrial's RIG have a smoother-colored tail, I honestly don't remember.

All of the alien AMI helmets now have transparent visors when inactive.

The Tajaran's AMI also has slightly larger arms, Tajaran-shaped boots based on the Tajaran Chief Engineer's suit, and a tail that's more smoothly colored.
...Although I have just realised that the standard AMI's boots poke through the sprite and make the effect not as distinctive, but I suppose a little bit better shaped is better than not at all.

image
(I fixed one or two pixel bugs after posting this image, but they're not very visible here, shrug.)
The tail also doesn't disappear from between their legs when the suit is inactive, I'm not sure if it did before, but it definitely doesn't now.

All the alien stealth rig helmets now have darkened visor slits when inactive.

The Unathi ninja suit's chest green-thing now darkens when the suit is inactive, when the suit is active it also looks a little less blurred-ish, also a Unathi's skin will no longer poke through the suit, folks, when testing suits, make sure the wearer is not black.

The Unathi stealth rig had its shoulder plates and chest light removed as I added those to the ninja suit with the intent of it being the ninja suit's quirk, optimistically they'll be replaced with their own unique body someday.

The Unathi stealth rig's helmet has been made to look more like the human stealth rig's helmet

image
image

File: html/changelogs/Fire-and-Glory - Fire-and-Glory-alien-hardsuits.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - tweak: "The Tajaran variant of the AMI has been modified following complaints of their feet being 'literrrally not shaped for the boots'."
+  - tweak: "The Unathi stealth rig looks a little less like a ninja suit and a bit more like the stealth rig. (didn't have the heart to do the Tajarn&Skrell rigs)"
+  - bugfix: "All Tajaran ERT helmets no longer permit the edge of the Tajaran's face to poke out of the helmet."
+  - bugfix: "The skin of the wearer will no longer poke through the Unathi ninja suit."
+  - tweak: "Other, more low-profile changes have been made to various alien RIGsuits."
+  - experiment: "Punted puppies."
```
Comment (Lohikar) [inline html/changelogs/Fire-and-Glory - Fire-and-Glory-alien-hardsuits.yml]: Nice.

Comment (Fire-and-Glory) [inline html/changelogs/Fire-and-Glory - Fire-and-Glory-alien-hardsuits.yml]: You gotta do what you gotta do.

**Pull request merged**